### PR TITLE
fix: add headers

### PR DESCRIPTION
**Pull request by joachimdekker**
This PR conforms to Gerard... Goodbye ZOMSA.

### CODE REVIEW

File: Aplib.Core/Belief/BeliefSets/BeliefSet.cs [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 namespace Aplib.Core.Belief.BeliefSets
 {
     /// <summary>
     /// This class can be inherited to define a set of beliefs for an agent.
-    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" /> 
+    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" />
     /// are automatically updated when calling <see cref="UpdateBeliefs" />.
     /// </summary>
-    public abstract class BeliefSet : IBeliefSet
+    public class BeliefSet : IBeliefSet
```
Comment (Tboefijn) [inline Aplib.Core/Belief/BeliefSets/BeliefSet.cs:15]: Does this need to be abstract?

**Review by Tboefijn — CHANGES_REQUESTED** [review]
Left a comment.

File: Aplib.Core/Belief/BeliefSets/BeliefSet.cs [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 using System.Linq;
 
 namespace Aplib.Core.Belief.BeliefSets
 {
     /// <summary>
     /// This class can be inherited to define a set of beliefs for an agent.
-    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" /> 
+    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" />
     /// are automatically updated when calling <see cref="UpdateBeliefs" />.
     /// </summary>
```
Comment (SilasPeters) [inline Aplib.Core/Belief/BeliefSets/BeliefSet.cs:14]: Interesting, what were your thoughts behind this?

**Review by SilasPeters — APPROVED** [review]

File: Aplib.Core/Belief/BeliefSets/BeliefSet.cs (hunk 1 above)
Comment (joachimdekker) [reply Aplib.Core/Belief/BeliefSets/BeliefSet.cs:15]: No, that was the sonar comment. There is no abstract function.

File: Aplib.Core/Belief/BeliefSets/BeliefSet.cs (hunk 2 above)
Comment (joachimdekker) [reply Aplib.Core/Belief/BeliefSets/BeliefSet.cs:14]: I have removed some invisible characters that were present in the code for some reason. No clue why they were there, this probably must be such a thing.

**Review by Tboefijn — APPROVED** [review]
LGTM!

File: Aplib.Core/Belief/BeliefSets/BeliefSet.cs (hunk 2 above)
Comment (SilasPeters) [reply Aplib.Core/Belief/BeliefSets/BeliefSet.cs:14]: Lol I responded to the wrong line. I wanted to react to the abstract thing haha, damn I was tired

Comment (SilasPeters) [reply Aplib.Core/Belief/BeliefSets/BeliefSet.cs:14]: Probably was whitespace at the end

**Pull request merged**